multinet/uploaders/swagger/newick.yaml

```diff
@@ -0,0 +1,28 @@
+Upload a Newick file to a collection of tables
+---
+consumes:
+  - text/plain
+
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: data
+    in: body
+    description: Raw Newick text
+    schema:
+      type: string
+      example: ((raccoon:19.19959,bear:6.80041):0.84600,((sea_lion:11.99700, seal:12.00300):7.52973,((monkey:100.85930,cat:47.14069):20.59201, weasel:18.87953):2.09460):3.87382,dog:25.46154);
+
+responses:
+  200:
+    description: Newick data uploaded to tables
+    schema:
+      type: object
+      properties:
+        edgecount:
+          type: integer
```

Comment:
In all the uploaders using this definition shows what would be 0 uploaded nodes and edges. Can we change these to match up with the example data and what you'd expect uploading that to a fresh workspace?

Reply:
The major reason I didn't add examples for these is that this response is almost certainly not what we should be sending back. A more useful response would be, for instance, the list of tables created, and what workspace they belong to. I think we should open an issue to discuss that, rather than spend the time on enriching these examples.

On the other hand, it's pretty simple to create examples for these too. Let me see about doing that as I respond to your review as a whole.

Reply:
50daf7efb8268f5f7a3970dcf620e9fec59df838